OSDOCS#18629: Update the z-stream RNs for 4.20.14 (#106337)
openshift/openshift-docs: modules/zstream-4-20-14.adoc

// Module included in the following assemblies:
//
// * release_notes/ocp-4-20-release-notes.adoc

:_mod-docs-content-type: REFERENCE
[id="zstream-4-20-14_{context}"]
= RHSA-2026:2119 - {product-title} {product-version}.14 fixed issues advisory

[role="_abstract"]
Issued: 11 February 2026

> {product-title} release {product-version}.14 is now available. The list of fixed issues that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2026:2119[RHSA-2119] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2026:2076[RHBA-2026:2076] advisory.

Reviewer: ```suggestion
{product-title} release {product-version}.14 is now available. The list of fixed issues that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2026:2119[RHSA-2026:2119] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2026:2076[RHBA-2026:2076] advisory.
```

* Before this update, the software catalog in the *Developer* view encountered an issue due to missing `Devfiles` data when a disconnected {product-title} cluster had the `Cluster Samples` Operator removed. As a consequence, users navigating to the *+Add* page encountered an empty software catalog with an error message. With this release, the  `Devfiles` data lookup in the software catalog is disabled, avoiding the error on the *+Add* page and resolving the `Error loading Catalog items` message. (link:https://issues.redhat.com/browse/OCPBUGS-74158[OCPBUGS-74158])

* Before this update, the upgrade to {product-title} 4.18 caused a loss of network connectivity for virtual machine (VM) pods using `ovn-k8s-cni-overlay` `localnet` network address translations (NAD). As a consequence, VM pod network connectivity was lost during the upgrade. With this release, the fix maintains network connectivity during cluster upgrades to {product-title} 4.18 for VMs using `ovn-k8s-cni-overlay` `localnet` NADs. As a result, the upgrade does not cause a loss of network connectivity. (link:https://issues.redhat.com/browse/OCPBUGS-74268[OCPBUGS-74268])

> * Before this update, static pods were terminated prematurely due to the `priorityClassName` field not recognized by the kubelet agent. As a consequence, premature termination of static pods led to long shutdown times and storage layer issues in {sno} environments. With this release, static pods acknowledge the `priorityClassName` field for shutdown order in {sno} environments. As a result, static pods follow shutdown order, reducing long shutdown times and storage layer issues. (link:https://issues.redhat.com/browse/OCPBUGS-74624[OCPBUGS-74624])

Reviewer: Seems this is missing a word, but could be flipped to avoid passive anyway:
```suggestion
* Before this update, static pods were terminated prematurely because the kubelet agent did not recognize the `priorityClassName` field. As a consequence, premature termination of static pods led to long shutdown times and storage layer issues in {sno} environments. With this release, static pods acknowledge the `priorityClassName` field for shutdown order in {sno} environments. As a result, static pods follow shutdown order, reducing long shutdown times and storage layer issues. (link:https://issues.redhat.com/browse/OCPBUGS-74624[OCPBUGS-74624])
```
